### ISSUE TITLE
Custom collection specified on a single instance appears also into reports folder of the other instances

### ISSUE THREAD

**Issue by acappelletti** [trimble-oss/dba-dash#785]
HI,
I don't know if this is correct behavior... but having defined "custom collections" only on a single connection string (1 out of 4) I find the "reports" folder present (but not valorized) in the other instances. 

Of course they are empty reports but I didn't expect them to appear in the other instances too.
![image](https://github.com/trimble-oss/dba-dash/assets/43847440/d0ecde1c-819a-42be-9e94-b87fc2a9a5ae)

Thanks Alen

**Comment by DavidWiseman** [comment]
Hi, this is how it's currently designed to work.  The custom report feature isn't linked to the custom collection.  It works like this:

The app looks for any stored procedure in the UserReport schema. The parameters present in the stored procedure are used to supply context from the application and control what levels in the tree the reports are visible.  
[More info here](https://dbadash.com/docs/how-to/create-custom-reports/)

You can create custom reports based on the data already collected by DBA Dash without a custom collection.  The app has no way of knowing that the custom report is linked to a custom collection that you have defined on certain instances.  

I do think it would be useful to have a mechanism to control the visibility of reports so they are only shown in places where they are applicable.  An option will likely be added for this at some point.

**Comment by acappelletti** [comment]
I agree with you on this "...I do think it would be useful to have a mechanism to control the visibility of reports so they are only shown in places where they are applicable. An option will likely be added for this at some point."

Thanks David!
